subjects/argsort/README.md

```diff
@@ -1,30 +1,27 @@
 ## argsort
 
-### Instruction
+### Instructions
 
 Write a program where you define if an argument is sorted or not sorted:
```

Comment:
Write a program that checks if an argument is sorted or not.

Reply:
done